src/components/wallet/TokenAmount.js

```diff
@@ -22,9 +25,9 @@ const formatToken = (amount, decimals) => {
     return removeTrailingZeros(formattedAmount)
```

Comment:
Just happened to notice this logic on L22/L23.  It looks like we are changing the value we're displaying in such a way that it is not actually an accurate number -- is this essentially performing a `round down` or a `.toFixed()` kind of operation via string manipulation?  If so, could we use BN.js for this instead so it is declarative rather than imperatively written?

If we need to leave it the way it is rather than use BN.js, It would be nice to have a comment on this, or (even better) move this logic into a named function so it is readily apparent that we are rounding the value or converting it to a fixed precision, without having to mentally parse the entire expression to understand what it is for, and have an explanation co-located with the code as to _why_ we need to transform the last digit a `1` here.

Reply:
If I remember correctly, that was the logic copied from `near-api-js` - I think we were planning to export this fn from `near-api-js` and use it here. But not sure :)